### DISCUSSION TITLE
메인 화면 구성 텍스트

### DISCUSSION THREAD

**Discussion by Jsueeee** [euphony-io/check-in-listener#29] [General]
@euphony-io/eu-us 
맨 아래 화면이 스피커 쪽 앱 화면입니다!
여기에 맞춰서 구성을 하고 있는데,
버튼 텍스트를 뭐로 할지 고민이네요!
일단 이렇게 '체크인 수신 시작' 이라고 했는데, 괜찮을까요? 
방문자 확인 시작이라던가... 음....
좀 더 좋은 아이디어 있으면 공유 부탁드립니다 ㅎㅎ

![image](https://user-images.githubusercontent.com/66951780/135703523-0e5cbb2b-bd3b-45ba-a764-7430f8c9dd10.png)

![image](https://user-images.githubusercontent.com/66951780/135703499-ef8f4c20-58d2-4d8c-9d91-f88042809b57.png)

**Comment by designe** [comment]
방문자 확인 시작도 좋네요!ㅋㅋㅋ

이용자 방문 확인 시작? ㅋㅋ
이런 네이밍이 중요한데 말이죠 ㅋㅋㅋㅋㅋ

**Comment by eunzinn** [comment]
방문자 확인 시작이 이용자 입장에서 좋을 것 같네요! ㅎㅎ

**Comment by YoungSeokHong** [comment]
방문자 확인 시작이 조금 더 정감 가네요 ㅋㅋㅋ
버튼을 클릭하기 전에는 '방문자 신호 수신', 버튼이 클릭 된 후에는 '신호 수신 중단'으로 하는 건 어떨까요??

**Comment by sujin9** [comment]
저도 방문자 확인 시작이 좋은 것 같아요!
영석님 말씀처럼 텍스트 바뀌는 것도 좋은 의견 같습니다! :)
